cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -345,4 +361,39 @@ def test_custom_button_open_url_evm_obj(setup_obj, button_group):
     Bugzilla:
         1550002
     """
-    pass
+    group, obj_type = button_group
+
+    button = group.buttons.create(
+        text=fauxfactory.gen_alphanumeric(start="btn", separator="_"),
+        hover=fauxfactory.gen_alphanumeric(),
+        open_url=True,
+        system="Request",
+        request=method.name,
+    )
+    request.addfinalizer(button.delete_if_exists)
+
+    view = navigate_to(setup_obj, "Details")
+    custom_button_group = Dropdown(view, group.hover)
+    assert custom_button_group.has_item(button.text)
+
+    # TODO: Move windows handling functionality to browser
+    initial_count = len(view.browser.selenium.window_handles)
```

Comment:
**Required** We shouldn't have selenium calls in a test case, this should be moved to the framework. I think that move should be part of this PR. Or this PR will be dependent on another PR addresses the framework enhancement.

Reply:
Hey, 
I was planning to do this latter as not that much necessary so added `TODO`. let handle this :+1: I wrapped selenium important method in `wt.core` Browser; so other team also take benefit of this.

Now this PR depends on: https://github.com/RedHatQE/widgetastic.core/pull/157

Reply:
@jawatts, do you mind if I merge it as is ?
The thing is that managing windows isn't implemented in widgetastic yet and requires some discussion.
From my point, widgetastic should have Window class instead of wrapping selenium calls.